VEKT2.CPP

```diff
@@ -19,103 +19,89 @@ vekt2 vekt2::operator-(vekt2 a) { return vekt2(x - a.x, y - a.y); }
 
 double vekt2::operator*(vekt2 a) { return x * a.x + y * a.y; }
 
-double vekt2::operator%(vekt2 b) { return x * b.y - y * b.x; }
-
-void vekt2::forgatas(double alfa) {
-    double a = sin(alfa);
-    double b = cos(alfa);
+void vekt2::rotate(double rotation) {
+    double a = sin(rotation);
+    double b = cos(rotation);
     double xo = x;
     x = b * x - a * y;
     y = a * xo + b * y;
 }
 
-double abs(vekt2 a) { return gyok(a.x * a.x + a.y * a.y); }
-
-double absnegyzet(vekt2 a) { return a.x * a.x + a.y * a.y; }
+double abs(vekt2 a) { return square_root(a.x * a.x + a.y * a.y); }
```

Comment:
Why is this method not length instead of abs? It calculates the length of a vector